civihr_default_theme/scss/components/_pivot-table.scss

```diff
@@ -2,18 +2,31 @@
 // --------------------------------------------------
 
 .pvtTable-civi {
+  table {
```

Comment:
I just noticed, this whole file does not follow BEM naming convention. I know this is a old file. 
@reneolivo could you check how much work we need to do to convert everything to BEM, and create proper re usable components. 
If the work is too much, I suggest to create a ticket under Technical improvements epic, so that we can pick it up later.